### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by phempshall** [phempshall/http-headers#13]
Add translations for more languages.

**Comment by c-sanchez** [comment]
Yeah that can be nice, I would like help with Spanish if you are agree :)

**Comment by phempshall** [comment]
Yes, thank you for the offer. 

I will first need to add support for localization. I will get back to you when it is ready. 

Cheers.

**Comment by c-sanchez** [comment]
Nice, let me know when is ready to make the translation :)
